yoga/YGNodePrint.cpp

```diff
@@ -90,7 +90,11 @@ static void appendEdges(
     const string& key,
     const YGStyle::Edges& edges) {
   if (areFourValuesEqual(edges)) {
-    appendNumberIfNotZero(base, key, edges[YGEdgeLeft]);
+    YGEdge edge = YGEdgeLeft;
+      if (edges[edge].isUndefined()) {
+          edge = YGEdgeAll;
+      }
+    appendNumberIfNotZero(base, key, edges[edge]);
```

Comment:
This could also be the combination of YGEdgeVertical or YGEdgeHorizontal. I think the robust way to do this would be to call one of the node edge resolution functions (`computeEdgeValueForColumn()` or `computeEdgeValueForRow()`)